samples/azure-quantum/hello-world/HW-rigetti-qiskit.ipynb

```diff
@@ -131,6 +131,7 @@
         "| --- | --- | --- | --- |\n",
         "| Rigetti QVM (simulator) | `rigetti.sim.qvm` | 20 qubits | Rigetti's cloud-based, [open-source](https://github.com/quil-lang/qvm) \"Quantum Virtual Machine\" simulator. Free to use. |\n",
         "| Aspen 11 (hardware) | `rigetti.qpu.aspen-11` | 40 qubits | Rigetti's \"Aspen-11\" universal, physical QPU. Read more at [Rigetti's website](https://qcs.rigetti.com/qpus). |\n",
+        "| Aspen M-2 (hardware) | `rigetti.qpu.aspen-m-2` | 80 qubits | Rigetti's "Aspen-M-2" universal, physical QPU. Read more at [Rigetti's website](https://qcs.rigetti.com/qpus). |\n",
```

Comment:
When I try to view the notebook, I see this: Invalid notebook - The Notebook Does Not Appear to Be Valid JSON.
By looking at the line above, I think the issue is unescaped quotation marks here.